app/routes/wow.export-search.tsx

```diff
@@ -118,7 +183,17 @@ const ExportSearch = () => {
         onClick={handleSubmit}
         error={error || itemName.error}
         loading={isSubmitting}
-        disabled={!itemId}>
+        disabled={!itemId}
+        action={getActionUrl(PAGE_URL, searchParams)}>
+        <div className="pt-2">
+          <div className="flex justify-end mb-2">
+            <SubmitButton
+              title="Share this search!"
+              onClick={handleCopyButton}
+              type="button"
+            />
+          </div>
+        </div>
         <div className="pt-3 flex flex-col">
           <DebouncedSelectInput
             title={'Item to search for'}
```

Comment:
Looks like there is missing a mapping of the itemID passed through the url params and this input box.

We probably want to set a default value if an itemID was passed through the loaderData.
Might need to update the debounce props to handle this case!